### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Picture in Picture Chrome Extension
+
+This extension demonstrates the Picture in Picture API in Chrome.
+
+**Note:** This only works in Chrome 68 or later.
+
+**To Install and Use It
+1. In `Chrome://flags` enable the following flags:
```
Comment (lgarron) [inline README.md]: Nit: lowercase `chrome`? ;-)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+   * \#enable-experimental-web-platform-features
+   * \#enable-surfaces-for-videos
+   * \#enable-picture-in-picture
+
+1. Download and extract the [extension zip file](https://github.com/beaufortfrancois/picture-in-picture-chrome-extension/archive/master.zip).
+1. In `Chrome://extensions` toggle **Developer mode** (upper-right corner) if it is not already on.
```
Comment (lgarron) [inline README.md]: ditto

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+1. To load the extension, click **Load Unpacked**.
+1. In the dialog box that appears locate and select the `src/` folder from the directory where you unpacked the zip file.
+1. Navigate to any YouTube video such as this one of [a band in Prague](https://www.youtube.com/watch?v=tbcnzyDQE1U).
```
Comment (lgarron) [inline README.md]: Mention that you have to click the extension icon to trigger PiP?
(It's fairly logical, but not self-explanatory.)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Picture in Picture Chrome Extension
+
+This extension demonstrates the Picture in Picture API in Chrome.
+
+**Note:** This only works in Chrome 68 or later.
+
+**To Install and Use It
```
Comment (japacible) [inline README.md]: Add closing **

**Comment by beaufortfrancois** [comment]
Thank you very much!

**Pull request merged**